src/ray/gcs/tables.cc

```diff
@@ -86,7 +96,12 @@ Status Log<ID, Data>::AppendAt(const JobID &job_id, const ID &id,
 template <typename ID, typename Data>
 Status Log<ID, Data>::Lookup(const JobID &job_id, const ID &id, const Callback &lookup) {
   num_lookups_++;
-  auto callback = [this, id, lookup](const std::string &data) {
+  const auto start_time = current_time_ms();
```

Comment:
It looks like that you missed adding the counter to `Add`.

Reply:
It doesn't matter. I'll add all metrics in next PR. This PR just show some example metrics.